ui/pages/confirm-send-token/confirm-send-token.container.js

```diff
@@ -28,7 +32,7 @@ const mapDispatchToProps = (dispatch) => {
         txParams: { from, to: tokenAddress, gas: gasLimit, gasPrice } = {},
       } = txData;
 
-      const to = getTokenValueParam(tokenData);
+      const address = getTokenValueParam(tokenData);
       const tokenAmountInDec = getTokenAddressParam(tokenData);
```

Comment:
@brad-decker Am I missing something here or are these reversed?

Reply:
i'm not sure what you mean by these?

Reply:
@brad-decker, @Gudahtt After a bit more digging, these are definitely switched and the reason its not causing a visible issue is that the `onEdit` method that this is called in is not passed down to `ConfirmTokenTransactionBase` and so the `edit` button never shows up for token send flows. I'm not sure if this was a desired change at some point? At the very least these should be switched back and then since I know this is a refactor and not meant to fix anything we should make a quick follow issue to investigate whether or not we want the edit button available in token send flows?

Reply:
> i'm not sure what you mean by these?

@brad-decker the consts are being assigned the wrong values -> `address = value` and `tokenAmountInDec = address`

Reply:
Right, but the goal of this PR is not to fix issues that predated the PR -- I think this should be elevated to an issue.

Reply:
Ok I'll write up an issue.

Reply:
Issue written up here: https://github.com/MetaMask/metamask-extension/issues/11317

Reply:
Huh, great find!

RE: whether we want the Edit button on token flows, that's a great question. This came up during a design sync a while back regarding the updated send flow designs, but this question wasn't fully resolved IIRC. Maybe it has been since then.

I'm pretty sure that we had originally meant token confirmations to be editable. It seems like a useful feature.